openedx/features/caliper_tracking/tests/expected/seq_goto.json

```diff
@@ -0,0 +1,46 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "NavigatedTo",
+  "actor": {
+    "id": "http://localhost:18000/u/honor",
+    "name": "honor",
+    "type": "Person"
+  },
+  "eventTime": "2018-12-17T12:41:36.236Z",
+  "extensions": {
+    "extra_fields": {
+      "accept_language": "en-US,en;q=0.9",
+      "agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.102 Safari/537.36",
+      "course_id": "course-v1:edX+DemoX+Demo_Course",
+      "event_source": "browser",
+      "event_type": "seq_goto",
+      "host": "e5d2c598af1a",
+      "ip": "172.18.0.1",
+      "name": "edx.ui.lms.sequence.tab_selected",
+      "org_id": "edX",
+      "page": "http://localhost:18000/courses/course-v1:edX+DemoX+Demo_Course/courseware/interactive_demonstrations/19a30717eff543078a5d94ae9d6c18a5/?child=first",
+      "path": "/event",
+      "session": "3a045173e7f98cc06e3e8bac4f2389f6",
+      "user_id": 6
+    }
+  },
+  "id": "urn:uuid:d4618c23-d612-4709-8d9a-478d87808067",
+  "object": {
+    "extensions": {
+      "current_tab": 7,
+      "id": "block-v1:edX+DemoX+Demo_Course+type@sequential+block@19a30717eff543078a5d94ae9d6c18a5",
+      "new": 1,
+      "old": 7,
+      "tab_count": 7,
+      "target_tab": 1,
+      "widget_placement": "top"
+    },
+    "id": "http://localhost:18000/courses/course-v1:edX+DemoX+Demo_Course/courseware/interactive_demonstrations/19a30717eff543078a5d94ae9d6c18a5/?child=first",
+    "type": "WebPage"
```

Comment:
`Page` is more appropriate as object type.

Reply:
Done in latest commit